### ISSUE TITLE
Wacca touch and led error

### ISSUE THREAD

**Issue by JustDaniMusic** [xiaopeng12138/WAC-VR#56]
I've downloaded the newest update of WacVR and since then the wacca led and touch does not work anymore. The wacca initialisation screen does give a success on wacca Led. But it does not light up in wacvr. And wacca touch gives a fail.

**Comment by xiaopeng12138** [comment]
Did you set [touch] enable = 1 in your xxxxtools.ini? And some PC will cashing the DLL file, I need the out put of both cmd window

**Comment by JustDaniDJ** [comment]
Yes, [touch] enable = 0 and i don't see special things happening in terminal. It just looks like the same when it worked.

**Comment by xiaopeng12138** [comment]
pls try the latest build again

**Comment by mechamonkey** [comment]
Same here, latest version
![image](https://github.com/xiaopeng12138/WACVR/assets/363523/1c81ef3f-280e-4cc9-8e07-597c5ee8017e)
